src/pages/project/project_wizard/ProjectDetailsForm.tsx

```diff
@@ -1,10 +1,95 @@
+  const { values, setFieldValue, setFieldTouched, touched, errors } = useFormikContext<SaveCristinProject>();
+  const thisIsRekProject = isRekProject(project);
 
-export const ProjectDetailsForm = () => {
   return (
     <ErrorBoundary>
-      <Box sx={{ bgcolor: 'secondary.light', padding: '1.75rem 1.25rem' }}>test 2</Box>
+      <Box sx={{ display: 'flex', flexDirection: 'column', gap: '1rem' }}>
+        <FormBox>
+          <Typography variant="h2">{t('project.coordinating_institution')}</Typography>
+          <Field name={ProjectFieldName.CoordinatingInstitution}>
+            {({ field }: FieldProps<ProjectOrganization>) => (
+              <OrganizationSearchField
+                disabled={thisIsRekProject}
+                label={t('project.coordinating_institution')}
+                onChange={(selectedInstitution) => {
+                  const selectedCoordinatingInstitution: ProjectOrganization = {
+                    type: 'Organization',
+                    id: selectedInstitution?.id ?? '',
+                    labels: selectedInstitution?.labels ?? {},
+                  };
+                  setFieldValue(field.name, selectedCoordinatingInstitution);
+                }}
+                errorMessage={
+                  touched.coordinatingInstitution?.id && !!errors.coordinatingInstitution?.id
+                    ? errors.coordinatingInstitution?.id
+                    : undefined
+                }
+                fieldInputProps={{
+                  ...field,
+                  onBlur: () => {
+                    setFieldTouched(`${field.name}.id`);
+                  },
+                }}
+                selectedValue={field.value}
+                customDataTestId={dataTestId.registrationWizard.description.projectForm.coordinatingInstitutionField}
+              />
+            )}
+          </Field>
+        </FormBox>
+        <FormBox>
+          <Typography variant="h2" sx={{ pb: '0.25rem' }}>
+            {t('project.form.category')}
+          </Typography>
+          <Field name={ProjectFieldName.Categories}>
+            {({ field, form: { setFieldValue } }: FieldProps<TypedLabel[]>) => (
+              <Autocomplete
+                options={cristinCategories}
+                multiple
+                data-testid={dataTestId.registrationWizard.description.projectForm.projectCategoryField}
+                getOptionLabel={(option) => getLanguageString(option.label)}
+                isOptionEqualToValue={(option, value) => option.type === value.type}
+                value={field.value}
+                onChange={(_, value) => setFieldValue(field.name, value)}
+                renderInput={(params) => (
+                  <TextField {...params} variant="filled" label={t('project.project_category')} />
+                )}
+              />
+            )}
+          </Field>
+        </FormBox>
+        <FormBox sx={{ gap: '0.5rem' }}>
+          <Typography variant="h2" sx={{ pb: '0.25rem' }}>
```

Comment:
igjen noe med inconsistencies på om det legges til `pb` eller ikke i disse. Vet ikke om tanken er at det egentlig skal være på alle, eller om det ikke skal være på noen?
```suggestion
          <Typography variant="h2">
```

Reply:
Tror det er fordi jeg har kopiert fra forskjellige plasser. Denne tab'en er bare satt sammen av felt som var spredt utover to sider tidligere.